Dockerfile.web

```diff
@@ -1,25 +1,9 @@
 FROM ocaml/opam:debian-12-ocaml-4.14@sha256:06d58a5cb1ab7875e8d94848102be43f6492e95320e8e9a9ecb9167654d0ee3f AS build
```

Comment:
We'd want to have this `Dockerfile.web` to be in sync with `Dockerfile`.

Reply:
I have updated `Dockerfile.web`